src/cycax/cycad/engines/cycax_part_freecad.py

```diff
@@ -224,43 +225,73 @@ def hole(
             cyl.Placement = App.Placement(Vector(x, y, z), App.Rotation(Vector(0, 1, 0), 270))
         return cyl
 
-    def render_to_png(self, target_path: Path):
-        """This method will be used for creating a png of 2 specific views of the object.
+
+    def render_to_png(self, view: str|None = None):
```

Comment:
`str|None`  there is also an Optional Type. Code wise I do not think there is a difference.  I have used Optional in most of the code base so for consistency we should pick one.

Reply:
Latest run of Ruff/Black changed all Optional to "|" syntax so I assume thats the way to go.